src/less/global.less

```diff
@@ -154,6 +154,24 @@
     src: url("../fonts/FuturaDemi.woff2") format("woff2"), url("../fonts/FuturaDemi.woff") format("woff");
 }
 
+/**
+*Z-indexes
+*/
+@action-list-z-index:                     100;
+@datepicker-z-index:                      100;
+@dropdown-z-index:                        100;
+@navigation-z-index:                      100;
+@progress-steps-z-index:                  100;
+@tabs-z-index:                            100;
+@skip-link-z-index:                      -100;
+@skip-link-active-z-index:               1000;
+@topbar-z-index:                         400;
+@sidebar-z-index:                        400;
+@sheet-z-index:                          500;
+@dialog-z-index:                         600;
+@toast-z-index:                          700;
+@tooltip-z-index:                        1100;
```

Comment:
@espenbakken could we reorder it sorting the variables by their "z-index" value, instead of alphabetically?
(e.g.
``` css
@....: 600;
@....: 400;
@....: 100;
@....: -100;
...
```
This way we would have a clearer idea of the z-index hierarchy, representing the layers, with the highest value at the top